.gitignore

```diff
@@ -1 +1,2 @@
 /target
+.env
```

Comment:
let's not added .env to .gitignore yet

tests are failing because it's missing